### PR TITLE
INSTA-9105: Adding Introduction & Getting Started sections

### PR DESCRIPTION
**Pull request by aprilwebster**
# Why

Work to address the usability and user experience in our [Instana REST API documentation](https://instana.github.io/openapi/). 

# What

### Screenshot of updated REST API doc (openapi)

Screenshot 2024-06-26 at 7 50 22 PM
Screenshot 2024-06-26 at 7 50 29 PM

# References

- [Jira Story: INSTA-9105 - Reorganize and clean up general documentation in the public Instana API](https://jsw.ibm.com/browse/INSTA-9105)
- [Jira Epic: INSTA-6158 - Improve Instana Public API - Phase 1](https://jsw.ibm.com/browse/INSTA-6158)

# Checklist

- Feature flags added?
  - [ ] In the code: `FeatureFlags.java` and `SimpleFeatureFlags.java`
  - [ ] To the dev config template: `<component>/dev/config.yaml.j2`
  - [ ] To the k8s config templates: `instanactl/configs/components/<component>.yaml.tpl`
  - [ ] A tracker item (Kanbanize or similar) has been added to remove all feature flags introduced in this PR in a later release, once the feature has been rolled out.
- Migrations necessary?
  - [ ] Added migrations to `instanactl/migrations`
  - [ ] Migration listed on the "Infrastructure Changes / Migrations" board for the next release
- API change?
  - [x] Documentation updated
- Integration tests needed?
  - [ ] Integration tests ran (comment the PR with `run integration tests` to have Jenkins run the integration tests)
- Database change?
  - [ ] New database tables are scoped by tenant unit (keys include `tenant_unit_id`.)
- User facing change?
  - [ ] Release Notes PR created and linked in references
- Documentation needs an update?
  - [ ] Docs PR created and linked in references

# Merge Guidance

❗ Please use a **squash merge** unless there is an explicit reason you need to use a different merge strategy (e.g. you are bringing in changes from a previous release branch, or you have specific changes in your branch that you would like to retain). This keeps our commit history clean, makes changes more atomic, and makes it easier to revert changes.

Screen Shot 2023-06-14 at 10 14 56 AM

### CODE REVIEW

**Review by thebignet — APPROVED** [review]
LGTM

**Review by phani-p3 — APPROVED** [review]
LGTM

File: spec/descriptions/info.md [hunk 1]
```diff
@@ -1,197 +1,38 @@
...
-  }
-]
-EOF
-```
-
-#### Limitations
-
-Adhere to the following rate limits for the trace web service:
+### Rate Limiting
+A rate limit is applied to API usage. Up to 5,000 calls per hour can be made. How many remaining calls can be made and when this call limit resets, can inspected via three headers that are part of the responses of the API server.
```
Comment (FarhadInstana) [inline spec/descriptions/info.md:19]: can **be** inspected

**Review by FarhadInstana — APPROVED** [review]
Looks good.  I also like the fact that those two "Agent REST APIs" are finally removed from here.

File: spec/descriptions/info.md (hunk 1 above)
Comment (aprilwebster) [reply spec/descriptions/info.md:19]: @FarhadInstana - I'm going to add that into my next set of revisions, thank you!

**Pull request merged**